test/erlfmt_SUITE.erl

```diff
@@ -1094,8 +1086,23 @@ snapshot_range_partial(_) ->
         "z()   ->  2.\n",
     % Only x() should be touched.
     Reference = "x() -> 0.\n",
-    Output = erlfmt:format_string_range(Original, {1, 1}, {1, 8}, []),
-    assert_snapshot_match(list_to_binary(Reference), Output).
+    Result = erlfmt:format_string_range_extract(Original, {1, 1}, {1, 8}, []),
+    assert_snapshot_match_range(Reference, Result).
+
+snapshot_range_partial_reinjected(_) ->
```

Comment:
Would be nice to include a few more tests for reinjecting that formats the middle one, only the final one, none of them, all of them, two of them.

Reply:
Done!